.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
The libpng api is pretty stable, this pull request just happens to use a newer feature to get the bitmap in one of the formats expected by libcar. I think it's fine to use the system provided version as long as it is 1.6+ which is common enough by now.